tests/src/main/java/io/apicurio/tests/utils/HttpUtils.java

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.apicurio.tests.utils;
+
+import io.apicurio.registry.ccompat.rest.RestConstants;
+import io.apicurio.registry.types.RuleType;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+
+import static io.restassured.RestAssured.given;
+
+public class HttpUtils {
```

Comment:
Why don't you just use the Registry client we already have impled?

Reply:
I didn't know we have it, I will change it :)